spec/mailers/volunteer_notifier_mailer_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe VolunteerNotifierMailer, type: :mailer do
+  describe "send_registration_email" do
+    VolunteerStub = Struct.new(:email, :name)
+    volunteer = VolunteerStub.new('thor@asgard.gov', 'Thor Odinson')
```

Comment:
How about instead of using a stub, we just use the real model:

`volunteer = Volunteer.new(email: 'thor@asgard.gov', name: 'Thor Odinson')`

Or since we don't test the email and name, we can use `FactoryBot`:

`volunteer = FactoryBot.build(:volunteer)`

I know it's nice to make sure we're only testing the Mailer here, but I'd like to stay aware from stubbing as much as possible.  My preference is to stub external services and complicated relationships that don't matter to the test at hand. But the less stubbing we have, the more we're actually testing the real system.

Reply:
Fair enough. I went with `Struct` in order to keep this divorced from any validation applied to `Volunteer`, but I don't have any strong objections to changing it